### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by SmsS4** [seaweedfs/seaweedfs#5145] [Q&A] [answered]
I'm using seaweedfs in our production and we really need some features of next release
Should i just build seaweedfs or wait for release? Any idea when will be the next release?

**Comment by sgonv** [comment]
How to upgrade to new version?

**Comment by kmlebedev** [comment, marked as answer]
You can always take the latest [dev](https://github.com/seaweedfs/seaweedfs/releases/tag/dev) version compiled from the master branch

**Reply by SmsS4** [reply]
nice thanks
